src/LabelStudio.tsx

```diff
@@ -1,55 +1,98 @@
+import { configure } from 'mobx';
+import { destroy } from 'mobx-state-tree';
 import { render, unmountComponentAtNode } from 'react-dom';
+import { toCamelCase } from 'strman';
+
+import { LabelStudio as LabelStudioReact } from './Component';
 import App from './components/App/App';
 import { configureStore } from './configureStore';
-import { LabelStudio as LabelStudioReact } from './Component';
-import { registerPanels } from './registerPanels';
-import { configure } from 'mobx';
-import { EventInvoker } from './utils/events';
 import legacyEvents from './core/External';
-import { toCamelCase } from 'strman';
-import { isDefined } from './utils/utilities';
 import { Hotkey } from './core/Hotkey';
 import defaultOptions from './defaultOptions';
-import { destroy } from 'mobx-state-tree';
 import { destroy as destroySharedStore } from './mixins/SharedChoiceStore/mixin';
-import { cleanDomAfterReact, findReactKey } from './utils/reactCleaner';
+import { EventInvoker } from './utils/events';
 import { FF_LSDV_4620_3_ML, isFF } from './utils/feature-flags';
+import { cleanDomAfterReact, findReactKey } from './utils/reactCleaner';
+import { isDefined } from './utils/utilities';
+
+declare global {
+  interface Window { Htx: any }
```

Comment:
do we want to continue exposing Htx? are we or any one actually using this? im wondering if we want to only expose this variable on a specific use case like while debugging. maybe we can look at only making Htx available if user has added debug=true to the url when loading the page. this'll help clean things up yet still allow us to use this when debugging. still entirely dependant on if Htx is used anywhere

Reply:
did a quick search, I do see that it is used with in editor, editor/e2e and refrenced in LSP ReviewPage. but now that we're moving to a mono-repo setup we maybe able to move away from this direction, open to discussion on this

Reply:
Thank you, that's important question! **tl;dr**: we need it
1. It's pretty common for libraries to expose their global(s) to access features from anywhere. There are different approaches of course, but that's not unusual and bad practice.
2. It really helped lot of times to deal and investigate issues — at any point we can get access to virtually any value from the console. We can ask anyone to do this at any time, especially in the middle of labeling without any reloading.
3. Unfortunately we use `Htx` in some fundamential places, the main one is detection of region type in [MST model Area](https://github.com/HumanSignal/label-studio-frontend/blob/f209200de05aad01cba5dffc59883d0de4615458/src/regions/Area.js#L55), plus couple of other places.
4. If we leave it anyway, but cover with some condition, this will leave the same code plus a little more. And I don't think this var is conflicting with anything. Tell me if you have objections and second thoughts!